data/strings/types_strings.txt

```diff
@@ -101,8 +101,9 @@
     zh-Hans = 缆车车站
     zh-Hant = 纜車站
 
-  [type.aeroway]
+[type.aeroway]
```

Comment:
Accidental shift?

Reply:
oh, yeah, sorry. Should I undo the shift ?